Added support for tokens and fixed a bug where the file name was longer that 260 chars it would throw an error (#34)
pranjalagg/spdl: README.md

This project is a tool to allow users to effortlessly download tracks and playlists fetched from Spotify, complete with metadata and album art.

> [!IMPORTANT]

> > Due to the new updates to the api, a token is needed every once in a while. Thus everytime prompted you'll have to provide the token.

Reviewer: ```suggestion
> Due to the new updates to the api, a token is needed every once in a while. Thus everytime prompted you'll have to provide the token. Get the token using the below steps:
```

> > 1. Go to https://spotifydown.com/ and open the network tab in the devtools (right click > ispect > navigate to the network tab)

Reviewer: ```suggestion
> 1. Go to https://spotifydown.com/ and open the "Network" tab in the browser devtools (right click > Inspect > navigate to the Network tab or use the key combination: Ctrl + Shift + I)
```

> > 2. While the network tab is open press download.

Reviewer: ```suggestion
> 2. While the Network tab is open press the download button.
```

> 3. Press download again. You will see some activity in the network tab.

> > 4. Filter to only show fetch requests and check through them for the one request url that has the token.

Reviewer: ```suggestion
> 4. Filter to only show `fetch` requests (from the "Type" column) and check through them for the one request url that has the token.
```
